django/contrib/admin/static/admin/css/forms.css

```diff
@@ -226,7 +226,7 @@ fieldset.collapsed h2 {
 }
 
 fieldset .collapse-toggle {
-    color: var(--body-bg);
+    color: var(--header-link-color);
```

Comment:
The only problem now is that it is now unclear whether it is a link or not as the color clashes with the normal text around it.

Reply:
I don't see an issue, it's exactly the same as in the light mode:
![image](https://user-images.githubusercontent.com/2865885/110334147-b93da380-8022-11eb-90b4-29e70e2bd5bb.png)

Reply:
Ok, so something that was off anyway. Perhaps something the a11y team can fix.

Reply:
I’m imagining we see a few more of these, but with vars in play it becomes a much simpler thing to address. So 💃